tests/zfs-tests/tests/functional/rsend/send_realloc_files.ksh

```diff
@@ -71,7 +71,7 @@ for i in {1..$passes}; do
 	# Randomly modify several dataset properties in order to generate
 	# more interesting incremental send streams.
 	rand_set_prop $POOL/fs checksum "off" "fletcher4" "sha256"
-	rand_set_prop $POOL/fs compression "off" "lzjb" "gzip" "lz4"
+	rand_set_prop $POOL/fs compression "off" "lzjb" "gzip" "lz4" "zstd"
```

Comment:
I noticed most tests use (something like) this and limit the levels to just:
"off" "lzjb" "gzip" "lz4" "zstd"

We could do the same with a little rewrite of "send-c_verify_ratio", which would improve the test duration by about 15% and leave room to add future compression algorithms without exeeding any duration.

It would also help keeping that test as close to master as possible.

Simply put:
There is no reason to test more than 1 of every compression algorithm, most tests don't do so either... Why should "send-c_verify_ratio"?

Reply:
**Solved**